openff/toolkit/tests/test_topology.py

```diff
@@ -319,6 +320,22 @@ def test_get_atom(self):
         with self.assertRaises(Exception) as context:
             topology_atom = topology.atom(8)
 
+    def test_topology_atom_element(self):
+        """Test getters of TopologyAtom element and atomic number"""
+        topology = Topology()
+        topology.add_molecule(self.toluene_from_sdf)
+
+        first_element = topology.atom(0).element
+        eighth_element = topology.atom(7).element
+
+        # Check if instances are correct
```

Comment:
(not blocking) This may be a little clearer

```suggestion
        # Check if types/instances are correct
```